packages/manager/tools/registry/README.md

```diff
@@ -51,6 +51,7 @@ Usage: manager-registry-dev [options] <fragmentsPath>
 Options:
   -V, --version                          output the version number
   -p, --port <port>                      server port (default: 8888)
+  -f, --filter <filter>                  filter fragment name (default: [])
```

Comment:
**thought**: maybe we can use `-e` (as expression like `grep`) or something different than `-f` as it could be a shortcut for `fallbackRegistry`

Reply:
option changed from `-f, --filter <filter>  ` to `-e, --regexp <pattern>`